source/more-info/statistics.markdown

```diff
@@ -3,6 +3,25 @@ title: "Expected data source not listed"
 description: "More information on if your expected data source is not listed."
 ---
 
-You're configuring a statistic but you couldn't find your source in the dropdown? That's caused by a bug in the integration providing the entity. Integrations need to configure their entities correctly so Home Assistant knows that we need to track statistics for it and how.
+You're configuring a statistic but you couldn't find your source in the dropdown?
+
+### Criteria for display
+
+* You are correctly collecting Long Term Statistics. Check [Developer Tools](https://www.home-assistant.io/docs/tools/dev-tools/) > Statistics for any issues. If not found, contact the integration provider.
```

Comment:
- Use `-` instead of `*` for lists
- Don't use full URLs, they don't work locally/dev/test/beta
- "contact the integration provider" -> That is confusing, people might actually contact their providers...